### PR TITLE
Subtitles changes and comments

### PR DESCRIPTION
**Pull request by yatil**
More comments in the pull request comments.

### CODE REVIEW

File: source/accessible-media/subtitles.html.erb.md [hunk 1]
```diff
@@ -1,6 +1,5 @@
 ---
-title: Subtitle concepts and examples
-nav_title: Subtitles
+title: Subtitles
```
Comment (yatil) [inline source/accessible-media/subtitles.html.erb.md:2]: Simplify title

File: source/accessible-media/subtitles.html.erb.md [hunk 2]
```diff
@@ -15,25 +14,11 @@ contributors:
 support: Developed with support from the <a href="https://www.w3.org/WAI/WCAGTA/">U.S. Access Board, WCAG TA Project</a>
 ---
 
-## Subtitle basics
```
Comment (yatil) [inline source/accessible-media/subtitles.html.erb.md:18]: Remove superfluous heading

File: source/accessible-media/subtitles.html.erb.md [hunk 3]
```diff
@@ -15,25 +14,11 @@ contributors:
...
-    (TTML)](https://www.w3.org/TR/ttaf1-dfxp/)
--   [Web Video Text Tracks (WebVTT)](https://w3c.github.io/webvtt/)
--   [SRT](https://matroska.org/technical/specs/subtitles/srt.html)
-
-And, as with captions, they can be open or closed:
-
--   **Closed subtitles** can be hidden and revealed by users, typically
-    by operating a button or menu on the player's control bar.
--   **Open subtitles** are visible to everyone and cannot be turned off.
+authoring tools and [formats as captions](formats.html). They can be provided in an [open or closed](index.html#open-closed) way.
```
Comment (yatil) [inline source/accessible-media/subtitles.html.erb.md:21]: Link to the captions formats and to a description of open/closed captions/audio descriptions/subtitles on the concepts page. (To be created.)

Removes repetition.

File: source/accessible-media/subtitles.html.erb.md [hunk 4]
```diff
@@ -15,25 +14,11 @@ contributors:
 support: Developed with support from the <a href="https://www.w3.org/WAI/WCAGTA/">U.S. Access Board, WCAG TA Project</a>
 ---
 
-## Subtitle basics
-
 Subtitles (also known as foreign-language subtitles) are, from a
```
Comment (yatil) [inline source/accessible-media/subtitles.html.erb.md:17]: “Subtitle” is a weird word. In Germany, the term for captions is “Untertitel für Hörgeschädigte,” literally translated “Subtitles for the hard of hearing”.  I don’t know if there are other forms of subtitles than foreign-language subtitles in English.

File: source/accessible-media/subtitles.html.erb.md [hunk 5]
```diff
@@ -44,33 +29,17 @@ Captions 1.0 (IMSC1)](https://www.w3.org/TR/ttml-imsc1/). Keep in mind that bitm
 ## Differences between subtitles and captions
 
 Subtitles are not the same as captions, nor are they aimed at the same
-audiences. Here are some of the basic differences.
+audiences. In contrast to captions, subtitles are…
```
Comment (yatil) [inline source/accessible-media/subtitles.html.erb.md:32]: Remove the table in favor for a simple four-item list.

File: source/accessible-media/subtitles.html.erb.md [hunk 6]
```diff
@@ -44,33 +29,17 @@ Captions 1.0 (IMSC1)](https://www.w3.org/TR/ttml-imsc1/). Keep in mind that bitm
...
-		<td>Contain non-speech information</td>
-	</tr><tr>
-		<td>Not positioned to convey who is speaking</td>
-		<td>Often positioned to indicate who is speaking</td>
-	</tr>
-</table>
+* … intended for people who can hear;
+* … translations of the audio;
+* … not containing non-speech information (indications of music, sound effects, etc.);
+* … not positioned to convey who is speaking.
```
Comment (yatil) [inline source/accessible-media/subtitles.html.erb.md:37]: I am not sure that we have this aspect (positioning the subtitle) covered in the captions tutorial. How would that work in a browser?

File: source/accessible-media/subtitles.html.erb.md [hunk 7]
```diff
@@ -80,8 +49,8 @@ the language in the `label` attribute:
 
 ~~~html
 <video controls>
-    <source src="myvideo.mp4" type="video/mp4" />
-    <track kind="subtitles" src="my_italian_subtitles.vtt" srclang="en" label="Italian" default />
+    <source src="myvideo.mp4" type="video/mp4">
+    <track kind="subtitles" src="my_italian_subtitles.vtt" srclang="en" label="Italian" default>
```
Comment (yatil) [inline source/accessible-media/subtitles.html.erb.md:53]: Again, more explaining what the individual attributes do would be nice.

File: source/accessible-media/subtitles.html.erb.md [hunk 8]
```diff
@@ -15,25 +14,11 @@ contributors:
...
--   [SRT](https://matroska.org/technical/specs/subtitles/srt.html)
-
-And, as with captions, they can be open or closed:
-
--   **Closed subtitles** can be hidden and revealed by users, typically
-    by operating a button or menu on the player's control bar.
--   **Open subtitles** are visible to everyone and cannot be turned off.
+authoring tools and [formats as captions](formats.html). They can be provided in an [open or closed](index.html#open-closed) way.
 
 In instances where subtitles cannot be presented using true text, they
```
Comment (yatil) [inline source/accessible-media/subtitles.html.erb.md:23]: This whole paragraph feels like a distraction:

> In instances where subtitles cannot be presented using true text, they
> can instead be presented as bitmaps: in other words, pictures of text or
> other symbols. The bitmaps themselves can still be transmitted using
> markup, though, by using [TTML Profiles for Internet Media Subtitles and
> Captions 1.0 (IMSC1)](https://www.w3.org/TR/ttml-imsc1/). Keep in mind that bitmap captions cannot be customized, which may diminish their usefulness to viewers with certain types of visual impairments.  Learn more about [why caption customization is important](production-captions.html#custom_cc).

1. Doesn’t this apply for captions as well?
2. Are bitmap subtitles a thing on the web?
3. Do we want to encourage people to use them or rather just show them the most optimal way?

**Review by yatil — COMMENTED** [review]
Also:

> In some cases, it may be necessary to use captions and subtitles
> together: for example, if an important sound effect is used in the audio
> track, that sound can be conveyed through a single sound-effect caption
> placed near the a subtitle, as shown in the image below.

When is this technique used? How does it work in HTML5? How do I author and display the position?

> \!\[A frame of video showing a sound-effect caption positioned above a
subtitle.](cc_sub_together.png)

Do we have the rights for this image? I think something schematic would be good as well.

File: source/accessible-media/subtitles.html.erb.md (hunk 4 above)
Comment (gfreedncam) [reply source/accessible-media/subtitles.html.erb.md:17]: see earlier comments elsewhere about confusion over terminology...

**Pull request closed without merging**